modules/RTC/BridgeChannel.js

```diff
@@ -170,6 +170,20 @@ export default class BridgeChannel {
     }
 
     /**
+     * Sends a "receiver video constraint" message via the channel.
+     * @param {Number} maxFrameHeightPixels the maximum fram height,
+     * in pixels, this receiver is willing to receive
+     */
+    sendReceiverVideoConstraintMessage(maxFrameHeightPixels) {
```

Comment:
Would you mind removing the "receiver" part from the message name? I can't imagine having a `SendVideoConstraintMessage`.